Update Immersive College of Winterhold

Sabre142 opened the pull request.

Sabre142 committed:
Update Immersive College of Winterhold

* Update mod entry
 - Add mod name.

* Add message
 - Third party patch available for
   The Gray Cowl Returns! (Creation Club)
   at https://www.nexusmods.com/skyrimspecialedition/mods/28687/

* Add tag suggestions
 - Actors.Perks.Add, adds new perks to actors in 0001C193,
   0001C194, 0001C195, 0001C196, 0001C197, 0001C198, 0001C199,
   0001C19A, 0001C19B, 0001C19C, 0001C19D, 0001C19E, 0001C19F,
   0001C1A0, 0001C23E, 0001E7D7
 - Actors.Perks.Remove, removes perks from actors in 0001C197,
   0001C199, 0001C19C, 0001C19D, 0001C1A0, 0001E7D7.
 - C.Regions, modifies the Cells Region
   subrecord in 00009170, 00009171.
 - ObjectBounds, changes the object bounds
   of things in 0001C193, 0001C19C.
 - NPC.AttackRace, changes the attack race
   of NPCs in 0001C19D, 0001C1A0.
 - NPC.DefaultOutfit, changes the default outfits of NPCs in 0001C193,
   0001C194, 0001C195, 0001C196, 0001C197, 0001C198,
   0001C199, 0001C19A, 0001C19B, 0001C19C, 0001C19D,
   0001C19E, 0001C1A0, 0001C23E, 0006A152.

* Remove tag suggestion
 - Names, does not modify the names of things, other than the
   forwards from USSEP in 0007C932, 0010F570.

* Add cleaning data
 - Version V6.2.

Sabre142 committed:
Update Immersive College of Winterhold

* Update mod entry
 - Add mod name.

* Add message
 - Third party patch available for
   The Gray Cowl Returns! (Creation Club)
   at https://www.nexusmods.com/skyrimspecialedition/mods/28687/

* Add tag suggestions
 - Actors.Perks.Add, adds new perks to actors in 0001C193,
   0001C194, 0001C195, 0001C196, 0001C197, 0001C198, 0001C199,
   0001C19A, 0001C19B, 0001C19C, 0001C19D, 0001C19E, 0001C19F,
   0001C1A0, 0001C23E, 0001E7D7
 - Actors.Perks.Remove, removes perks from actors in 0001C197,
   0001C199, 0001C19C, 0001C19D, 0001C1A0, 0001E7D7.
 - C.Regions, modifies the Cells Region
   subrecord in 00009170, 00009171.
 - ObjectBounds, changes the object bounds
   of things in 0001C193, 0001C19C.
 - NPC.AttackRace, changes the attack race
   of NPCs in 0001C19D, 0001C1A0.
 - NPC.DefaultOutfit, changes the default outfits of NPCs in 0001C193,
   0001C194, 0001C195, 0001C196, 0001C197, 0001C198,
   0001C199, 0001C19A, 0001C19B, 0001C19C, 0001C19D,
   0001C19E, 0001C1A0, 0001C23E, 0006A152.

* Remove tag suggestion
 - Names, does not modify the names of things, other than the
   forwards from USSEP in 0007C932, 0010F570.

* Add cleaning data
 - Version V6.2.

BellCubeDev commented on masterlist.yaml:
```diff
@@ -17695,35 +17697,43 @@ plugins:
       - Outfits.Add
       - Stats
       - Text
-    dirty:
```
From my (honestly quite limited) understanding, it doesn't seem like removing old CRC entries is a good idea. If a user chooses not to update (or misses it entirely), the dirty flag would, all of a sudden, no longer appear.

Just something for consideration.

sibir-ine replied:
Keeping old cleaning data leads to clutter with diminishing utility, so we tend to just replace it unless users have reason to continue to use an older version (e.g., the author officially supports it).

Sabre142 replied:
Pre-Version 6.2 downloads of the mod are not available anymore anyway, and it has been that way for almost a year.

BellCubeDev replied:
> Keeping old cleaning data leads to clutter with diminishing utility, so we tend to just replace it unless users have reason to continue to use an older version (e.g., the author officially supports it).

Cool, got it. I'll keep that in mind.

Sabre142 committed:
Add JK's Temple of Talos (#2206)

* Add mod entry
 - To the Location - Overhauls section.

* Add location
 - https://www.nexusmods.com/skyrimspecialedition/mods/56971/

* Add messages
 - For third party patches available at
   https://www.nexusmods.com/skyrimspecialedition/mods/35910/
   for various mods, including:
 - AI Overhaul.
 - Armor and Clothing Extension.
 - Cheesemod for EVERYONE.
 - Cloaks of Skyrim.
 - ESO Imports.
 - Enhanced Lights and FX.
 - Interesting NPCs.
 - Oblivion Artifacts Pack.
 - Palaces and Castles Enhanced.
 - Snazzy Furniture and Clutter Overhaul.
 - Solitude and Temple Frescoes 2019.
 - Unofficial Skyrim Special Edition Patch.

* Add cleaning data
 - Version 1.0.0: 0xDF26DC5A.

sibir-ine commented on masterlist.yaml:
```diff
@@ -17695,35 +17697,43 @@ plugins:
+        subs:
+          - 'The Gray Cowl Returns! (Creation Club)'
+          - '[Immersive College of Winterhold - More Patches](https://www.nexusmods.com/skyrimspecialedition/mods/28687/)'
+        condition: 'active("ccbgssse020-graycowl.esl") and not active("ICOW_GrayCowlCC_Patch.esp")'
```
These `*patch3rdParty` messages should be sorted above the other messages.

sibir-ine commented on masterlist.yaml:
```diff
@@ -17695,35 +17697,43 @@ plugins:
+      - NPC.AttackRace
       - NPC.Class
+      - NPC.DefaultOutfit
+      - ObjectBounds
```
The object bound edits appear to be CK noise & likely don't serve a purpose in-game.

```suggestion
```

Sabre142 committed:
Update Immersive College of Winterhold

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update Immersive College of Winterhold

* Move patch3rdParty messages above the other messages.

Co-Authored-By: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Merge branch 'ICOW' of https://github.com/Sabre142/skyrimse into ICOW

Sabre142 committed:
Revert "Add JK's Temple of Talos (#2206)"

This reverts commit 38aa8fcd1ff685e157b22d4416d491b0ce70ecf3.

Sabre142 committed:
Revert "Update Immersive College of Winterhold"

This reverts commit a4d3cb99e025540e247efd386b5ebc4192efa441.

Sabre142 committed:
Update Immersive College of Winterhold

* Remove ObjectBounds tag.
